proxy/common/_scm_version.pyi

```diff
@@ -1,3 +1,6 @@
 # This stub file is necessary because `_scm_version.py`
 # autogenerated on build and absent on mypy checks time
+from typing import Tuple, Union
+
 version: str
```

Comment:
I see what you meant by tuple bit earlier.  It worked within Makefile target then because it simply resulted in creation of `_scm_version.py` file :)